pkgs/top-level/all-packages.nix

```diff
@@ -9078,7 +9078,13 @@ in
 
   ocaml-crunch = ocamlPackages.crunch.bin;
 
-  ocamlformat = callPackage ../development/tools/ocaml/ocamlformat { };
+  ocamlformat_0_11_0 = callPackage ../development/tools/ocaml/ocamlformat/0.11.0.nix { };
```

Comment:
Do we really need all these versions though and when do get old versions removed again?

Reply:
cc @vbgl

Reply:
We have a script that is scraping OCamlformat configs on Github: https://gist.github.com/Julow/110dc94308d6078225e0665e3eccd433

Here are the most used versions as of now (the script may miss some projects and a lot of project don't declare the version they use):
```
  28 version = 0.14.1
  26 version = 0.13.0
  21 version = 0.12
  16 version = 0.11.0
  12 version = 0.14.2
   7 version = 0.9
   5 version = 0.14.0
```

0.11.0 is still used by a lot of projects but it is reasonable to remove it if less is better.